example/notifications/src/DialogLaunchController.js

```diff
@@ -44,30 +44,31 @@ define(
              periodically with the progress of an ongoing process.
              */
             $scope.launchProgress = function (knownProgress) {
-                var model = {
-                    title: "Progress Dialog Example",
-                    progress: 0,
-                    hint: "Do not navigate away from this page or close this browser tab while this operation is in progress.",
-                    actionText: "Calculating...",
-                    unknownProgress: !knownProgress,
-                    unknownDuration: false,
-                    severity: "info",
-                    options: [
-                        {
-                            label: "Cancel Operation",
-                            callback: function () {
-                                $log.debug("Operation cancelled");
-                                dialogService.dismiss();
+                var dialog,
+                    model = {
```

Comment:
Note for future, i believe we've changed our style guide regarding combined view statements so this doesn't need to be such a big diff when you have to indent the entire object literal just to combine var statements.

That said, I used the disable whitespace diff option (append ?w=0 to url e.g. https://github.com/nasa/openmct/pull/1063/files?w=0) which nicely hides this.